iot_packet_verifier/src/lib.rs

```diff
@@ -0,0 +1,127 @@
+use futures::stream::StreamExt;
+use slog::{self, info, o, Drain, Level, Logger};
+use std::collections::HashMap;
+//use std::io::{ErrorKind, Result};
+//use tokio_stream::StreamExt;
+// use tokio::sync::mpsc;
+
+use helium_proto::services::router::PacketRouterPacketReportV1;
+use helium_proto::Message;
+use poc_store::{file_source, Result};
+// FIXME for writing to S3:
+//use poc_store::FileStore;
+
+// Inputs:
```

Comment:
This should be PublicKey from helium_crypto?

Reply:
Fixed in commit e0e1c6a.  Thank you for that!